src/reducers/rootReducer.js

```diff
@@ -1,8 +1,8 @@
 import { combineReducers } from 'redux';
+import userReducer from './userReducer';
 
 const rootReducer = combineReducers({
-// dummy object for reducer
-  user: () => ({}),
+  user: userReducer,
```

Comment:
Could you look into using the property value shorthand? More like:

import user from './userReducer';

const rootReducer = combineReducers(
{
user,
}
);

More details about this ES6 style guide: https://eslint.org/docs/rules/object-shorthand.html